burotel/indico_burotel/plugin.py

```diff
@@ -0,0 +1,76 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from datetime import datetime, time
+
+from flask import current_app, json, redirect, request
+from marshmallow import Schema, fields
+from werkzeug.datastructures import ImmutableMultiDict
+
+from indico.core.plugins import IndicoPlugin
+from indico.util.marshmallow import NaiveDateTime
+from indico.web.flask.util import url_for
+from indico_burotel.controllers import WPBurotelBase
+
+
+class DateSchema(Schema):
+    start_dt = fields.Date()
+    end_dt = fields.Date()
+
+
+class DateTimeSchema(Schema):
+    start_dt = NaiveDateTime()
+    end_dt = NaiveDateTime()
+
+
+def _fix_unicode(data):
+    return {k: unicode(v) for k, v in data.viewitems()}
```

Comment:
why do we get non-unicode strings in the first place? :o

Reply:
Good question.

Reply:
Now I remember, it's marshmallow, which serializes the dates to UTF8.

Reply:
yuck, can we change/fix that?

Reply:
Doesn't look promising... https://github.com/marshmallow-code/marshmallow/blob/dev/marshmallow/fields.py#L1003